runtime/cocos/src/lib.rs

```diff
@@ -194,7 +194,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("regionx-parachain"),
 	impl_name: create_runtime_str!("regionx-parachain"),
 	authoring_version: 1,
-	spec_version: 2_002_000,
+	spec_version: 2_003_000,
```

Comment:
```suggestion
	spec_version: 2_002_000,
```
I will update this in a new PR since I want to make one more slight modification.